Libraries/Alert/RCTAlertManager.ios.js

```diff
@@ -10,6 +10,6 @@
 
 'use strict';
 
-const RCTAlertManager = require('../BatchedBridge/NativeModules').AlertManager;
+const RCTAlertManager = require('./NativeAlertManager');
```

Comment:
@sasurau4,
Could you update its call-sites inside [Alert.js](https://github.com/facebook/react-native/blob/master/Libraries/Alert/Alert.js#L13-L14), pls?

Also, this is a minor, but should we use `import RCTAlertManager from './NativeAlertManager'` here instead? I saw many PRs replacing it to this format.